### DISCUSSION TITLE
Anyone have this error when login into the admin panel, the sidebar is empty then the log have those error in production server. pls help

### DISCUSSION THREAD

**Discussion by duychip0909** [filamentphp/filament#6629] [Help]
![Untitled](https://github.com/filamentphp/filament/assets/91042370/e09a0f41-987d-4d4e-90f7-d0a0b38028da)

I have public all the things that need like livewire config file filament config file, change the asset url in livewire config file

**Comment by ArtMin96** [comment]
```cmd
npm run build
```
or
[Fix Filament assets not loading in Nginx](https://filamentphp.com/tricks/fix-filament-assets-nginx)

**Reply by duychip0909** [reply]
i've run build in ci cd but it's not working :<

**Reply by duychip0909** [reply]
i use cloudflare for setting up that source

**Reply by duychip0909** [reply]
and my issues is the alpinejs and seems the livewire is not loading into the admin panel

**Comment by danharrin** [comment]
There is no need to publish anything, if you server (specifically nginx) was configured correctly, then there would be no issue loading scripts

**Reply by duychip0909** [reply]
can you tell me about the basic config or setup when deploying TALL sources with filament on production server? how many step i have to do

**Reply by danharrin** [reply]
Deploying your site with Laravel Forge or Vapor will make it work first try, every time. Otherwise, check the Laravel docs https://laravel.com/docs/10.x/deployment#server-configuration
